src/main/java/net/neoforged/neoforge/common/Tags.java

```diff
@@ -35,8 +35,28 @@ public static class Blocks {
          * This is patched into the following method: {@link net.minecraft.world.entity.monster.EnderMan.EndermanLeaveBlockGoal#canPlaceBlock(Level, BlockPos, BlockState, BlockState, BlockState, BlockPos)}
          */
         public static final TagKey<Block> ENDERMAN_PLACE_ON_BLACKLIST = neoforgeTag("enderman_place_on_blacklist");
+
+        /**
+         * For denoting blocks that need tools that are Wood or higher to mine.
+         * By default, this is not added to any Minecraft tag since by default, Wood is in the lowest "tier".
```

Comment:
Per @Shadows-of-Fire:
```suggestion
         * By default, this is not added to any Minecraft tag since Wood is in the lowest "tier".
```